### PR TITLE
Remove references to ChiTech, part 2

### PR DESCRIPTION
**Pull request by andrsd**

### CODE REVIEW

**Comment by andrsd** [comment]
I will wait until #91 is merged (there will be a conflict), so I will fix that then. #91 arrived first, so should go in first... 
But feel free to look at this PR when you get time.

**Review by wdhawkins — APPROVED** [review]
Great to see these changes!

**Review by ragusa — APPROVED** [review]

File: framework/runtime.h [hunk 1]
```diff
@@ -56,166 +58,151 @@ extern MPI_Info& mpi;
...
 
-/**General utilities in ChiTech*/
-class Chi
-{
-public:
-  /** Global stack of handlers */
-  static std::vector<MeshHandlerPtr> meshhandler_stack;
-  static int current_mesh_handler;
+/** Global stack of handlers */
+extern std::vector<MeshHandlerPtr> meshhandler_stack;
```
[the remark this answers is not in the document]
Comment (andrsd) [reply framework/runtime.h:62]: It is like a forward declaration for classes. It says the compiler what type this is (so code can be compiled), but that the symbols will be defined elsewhere.

[the remark this answers is not in the document]
Comment (ragusa) [reply framework/runtime.h:62]: like in fortran's "EXTERNAL" ?

[the remark this answers is not in the document]
Comment (andrsd) [reply framework/runtime.h:62]: I don't know FORTRAN that well, so IDK.

**Comment by andrsd** [comment]
This is now rebased on latest `main`.

**Review by zhardy-lanl — APPROVED** [review]

**Pull request merged**